workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/MDFRequestForm/steps/Activities/Activities.tsx

```diff
@@ -119,6 +119,7 @@ const Activities = ({
 
 	const onRemove = (index: number) => {
 		setFieldValue(`activities[${index}].removed`, true);
+		arrayHelpers.remove(index);
```

Comment:
if order is importnat, we should have an extra line break in between. If order is not important then we should alphabetize.

Reply:
Done!